README.md

```diff
@@ -7,6 +7,10 @@
 > **already have full access** to the EIAP Ecosystem.
 > If you do not have this access, **please do not proceed.**
 
+>**Note:**
+If you need help accessing the EIAP Ecosystem,contact support 
```

Comment:
Nice to have: space after comma
But if the deadline is tight, this comment can be ignored

Reply:
addressed comments